src/Backend/WASMWrapper.js

```diff
@@ -145,4 +145,28 @@ export default class WASMWrapper {
 
     //     return DataSeriesFactory.getYValues(xs, boundStepFuntion)
     // }
+
+    async getSingleValue(program, ion, material, energy){
+        const wasm = await this.wasm()
+
+        const buf = wasm._malloc(Int32Array.BYTES_PER_ELEMENT)
+        const heap = new Uint8Array(wasm.HEAP32.buffer, buf, Int32Array.BYTES_PER_ELEMENT)
+
+        const res =  wasm.ccall('dedx_get_simple_stp','number',['number', 'number', 'number', 'number'],[ion,material,energy, heap.byteOffset])
+
+        const err =  new Int32Array(heap.buffer, heap.byteOffset, 1)[0]
+        
+        if(err !== 0){
+            console.error(`Dedx execution error ${err}`)
+            return NaN
+        }
+            
+
+        return res
+
+    }
+
+    async generateDefaults({program, ion, material}){
+        return [1,2,5,10,20,50,100]
```

Comment:
```suggestion
        return [1,2,5,10,20,50,100, 200, 500, 1000, 2000, 5000]
```

Reply:
But this is supposed to come from dedx and be dependet on the program we choose, right?

Reply:
Right, but as this is not done yet, I proposed a bit more realistic values